overlord/snapstate/handlers.go

```diff
@@ -113,7 +113,22 @@ func snapSetupAndState(t *state.Task) (*SnapSetup, *SnapState, error) {
 */
 
 const defaultCoreSnapName = "core"
-const defaultBaseSnapsChannel = "stable"
+
+func defaultBaseSnapsChannel() string {
+	channel := os.Getenv("SNAPD_BASES_CHANNEL")
```

Comment:
Could we document those someplace. This will be forgotten next week.

Reply:
they are for our own testing tough, not for users

Reply:
That doesn't preclude those from being documented. It's a hidden feature really.

Reply:
We don't have a place atm where we document this stuff (I only see SNAPD_DEBUG documented in HACKING.md and that's it) , they seem the wrong ones to start if we document only these.

Reply:
Traditionally those are documented on the manual page of the command that respects / uses them. I did this for snap-confine but we have a pretty poor manual page for snap/snapd for now.